pkgs/applications/emulators/box86/default.nix

```diff
@@ -0,0 +1,108 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gitUpdater,
+  cmake,
+  python3,
+  withDynarec ? (
+    stdenv.hostPlatform.isAarch32
+  ),
```

Comment:
This was made by the formatter?

Reply:
No, sorry. It did that in `box64` since that had a list of `||`'d options here, and I didn't re-run the formatter here after reducing this. Let me fix this.